Gemfile.lock

```diff
@@ -56,58 +56,54 @@ GEM
       jekyll (>= 3.7, < 5.0)
     jekyll-watch (2.2.1)
       listen (~> 3.0)
-    json (2.6.3)
+    json (2.7.2)
     kramdown (2.4.0)
       rexml
     kramdown-parser-gfm (1.1.0)
       kramdown (~> 2.0)
     liquid (4.0.4)
     liquid-c (4.0.1)
       liquid (>= 3.0.0)
-    listen (3.8.0)
+    listen (3.9.0)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
     mercenary (0.4.0)
-    nokogiri (1.15.4-arm64-darwin)
+    nokogiri (1.16.5-arm64-darwin)
       racc (~> 1.4)
-    nokogiri (1.15.4-x86_64-linux)
-      racc (~> 1.4)
-    parallel (1.23.0)
+    parallel (1.24.0)
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
-    public_suffix (5.0.3)
-    racc (1.7.1)
+    public_suffix (5.0.5)
+    racc (1.8.0)
     rainbow (3.1.1)
-    rake (13.0.6)
+    rake (13.2.1)
     rb-fsevent (0.11.2)
-    rb-inotify (0.10.1)
+    rb-inotify (0.11.1)
       ffi (~> 1.0)
-    rexml (3.2.6)
-    rouge (4.1.3)
-    rspec-core (3.12.2)
-      rspec-support (~> 3.12.0)
-    rspec-expectations (3.12.3)
+    rexml (3.2.8)
+      strscan (>= 3.0.9)
+    rouge (4.2.1)
+    rspec-core (3.13.0)
+      rspec-support (~> 3.13.0)
+    rspec-expectations (3.13.0)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.12.0)
-    rspec-support (3.12.1)
+      rspec-support (~> 3.13.0)
+    rspec-support (3.13.1)
     safe_yaml (1.0.5)
-    sass-embedded (1.69.3)
-      google-protobuf (~> 3.23)
-      rake (>= 13.0.0)
+    sass-embedded (1.77.2-arm64-darwin)
+      google-protobuf (>= 3.25, < 5.0)
+    strscan (3.1.0)
     terminal-table (3.0.2)
       unicode-display_width (>= 1.1.1, < 3)
-    typhoeus (1.4.0)
+    typhoeus (1.4.1)
       ethon (>= 0.9.0)
     unicode-display_width (2.5.0)
     webrick (1.8.1)
     yell (2.2.2)
-    zeitwerk (2.6.12)
+    zeitwerk (2.6.14)
 
 PLATFORMS
-  arm64-darwin-21
-  arm64-darwin-22
```

Comment:
My local build adds `arm64-darwin-22` to the gemfile.lock. We originally added this ininhttps://github.com/uswds/uswds-site/pull/2577. Can we restore that here?

Reply:
Restored in[35a3e64](https://github.com/uswds/uswds-site/pull/2673/commits/35a3e64cece2e522ec5d9c7effa9a559870c2b45)!

Reply:
Was this added via the `bundle lock --add-platform PLATFORM` command? 

Worried about it being overridden again.

Reply:
It was but I think the only reason it was originally removed was because I had played with the bundler and ruby versions. If I recall correctly, this normally isn't overridden during regular dependency updates.

Reply:
Okay, thanks for clarifying.